lib/pure/collections/deques.nim

```diff
@@ -7,25 +7,24 @@
 #    distribution, for details about the copyright.
 #
 
-## Implementation of a `deque`:idx: (double-ended queue).
-## The underlying implementation uses a ``seq``.
+## An implementation of a `deque`:idx: (double-ended queue).
+## The underlying implementation uses a `seq`.
 ##
-## None of the procs that get an individual value from the deque can be used
+## Note that none of the procs that get an individual value from the deque should be used
 ## on an empty deque.
-## If compiled with `boundChecks` option, those procs will raise an `IndexDefect`
+## If compiled with the `boundChecks` option, those procs will raise an `IndexDefect`
 ## on such access. This should not be relied upon, as `-d:danger` or `--checks:off` will
-## disable those checks and may return garbage or crash the program.
+## disable those checks and then the procs may return garbage or crash the program.
 ##
 ## As such, a check to see if the deque is empty is needed before any
 ## access, unless your program logic guarantees it indirectly.
 
 runnableExamples:
-  var a = initDeque[int]()
+  var a = [10, 20, 30, 40].toDeque
 
-  doAssertRaises(IndexDefect, echo a[0])
+  doAssertRaises(IndexDefect, echo a[4])
 
-  for i in 1 .. 5:
-    a.addLast(10*i)
+  a.addLast(50)
   assert $a == "[10, 20, 30, 40, 50]"
```

Comment:
`assert a.toSeq == [10, 20, 30, 40, 50]` or equivalent (and keep just 1 line with `$`, eg `assert $a == "[22, 11, 20]"`)